docs: some improvement (#620)
anatawa12/AvatarOptimizer: .docs/content/docs/reference/trace-and-optimize/index.md

This component should be added to an avatar root. (Kind: [Avatar Global Component](../../component-kind/avatar-global-components))

{{< hint info >}}

Trace and Optimize is quite carefully designed under the premise that "**never let it affect the appearance**."\
So, if any problems occur, such as appearance is affected or some gimmick stops working, they are all caused by bugs in AAO, without exception.\

> Therefore, if you encounter any problems with this component, please report it and it will most likely be fixed.

Author: ```suggestion
Therefore, if you encounter any problems with this component, please report it.\
We will fix it as much as we can.
```
